tests/integration/states/test_pkg.py

```diff
@@ -853,18 +855,20 @@ def test_pkg_cap_002_already_installed(self, grains=None):
         self.assertFalse(version)
         self.assertFalse(realver)
 
-        # install the package already
-        ret = self.run_state('pkg.installed', name=realpkg, refresh=False)
+        try:
+            # install the package already
+            ret = self.run_state('pkg.installed', name=realpkg, refresh=False)
```

Comment:
should we check this response for a failure?

Reply:
I don't think it's 100% necessary. The aim of the two state runs here is to check that we get a no-op result the second time through, since the install should have passed the first time through and installed the package.

It may help in troubleshooting if the first install fails though, so I went ahead and added it :+1: